ts/Stock/Indicators/MACD/MACDIndicator.ts

```diff
@@ -374,7 +374,8 @@ class MACDIndicator extends SMAIndicator {
         series: TLinkedSeries,
         params: MACDParamsOptions
     ): (IndicatorValuesObject<TLinkedSeries>|undefined) {
-        var j = 0,
+        var indexToShift: number = (params.longPeriod as any) - (params.shortPeriod as any), // #14197
```

Comment:
isn't it better to have `as number` ?

Reply:
`params.longPeriod` type everywhere is declared as `any`
I think the convention is to leave types of previously declared variable that during the TS refactoring they all might be changed together to better types.